### ISSUE TITLE
Show token art if not default token

### ISSUE THREAD

**Issue by jknolte** [Eriku33/Foundry-VTT-Image-Hover#68]
Is it possible to add a setting here that shows character art unless the token portrait is not the default portrait? It would sort of be like the wildcard option.
![image](https://github.com/user-attachments/assets/346d0831-5ad3-45ba-8c84-1be067187c2c)

This will come in handy when using modules like Token Variant Art. Right now if I change a token to variant art, I'd rather have it show the token image than the character art.

**Comment by dineshm72** [comment]
Yes, this is also handy when using TVA to change the token art for creatures who can disguise themselves, like doppelgangers.  As is currently, after you change the token art you also have to change the hover image to match it manually.

**Comment by dineshm72** [comment]
Ahh, I had made a similar request already: https://github.com/Eriku33/Foundry-VTT-Image-Hover/issues/60
